internal/api/front/perses/src/app/project/dashboard/dashboard-details/dashboard-details.component.html

```diff
@@ -1 +1,33 @@
-<p>dashboard-details works!</p>
+<app-page>
+  <app-page-header>
+    <mat-spinner *ngIf="isLoading"></mat-spinner>
+    <h1 *ngIf="!isLoading">{{dashboard.metadata.name}}</h1>
+  </app-page-header>
+
+  <app-page-content>
+    <mat-spinner *ngIf="isLoading"></mat-spinner>
+    <app-page-sub-content *ngIf="!isLoading">
+      <ng-container *ngFor="let section of dashboard.spec.sections">
+        <h2>{{section.name}}</h2>
+        <div *ngFor="let panel of section.panels">
+          <h3>{{panel.name}}</h3>
+          <div>
+            <ngx-charts-line-chart
+            [view]="view"
+            [scheme]="colorScheme"
+            [results]="ngxData.get(section.name+'_'+panel.name)"
```

Comment:
In my opinion the compute of the name from section an panel should be in a function 
I guess that this function can be used elsewhere, maybe in the typescript code ?

Reply:
yeah it would be a bit more elegant if you are calling a public method from your component that is wrapping this thing. Like: 

```typescript

public getSomething(sectionName: string, panelName: string) {
  return this.ngxData.get(`${sectionName}_${panelName}`)
}
```